ipn/ipnlocal/local.go

```diff
@@ -6455,6 +6458,29 @@ func (b *LocalBackend) FileTargets() ([]*apitype.FileTarget, error) {
 	return ret, nil
 }
 
+func (b *LocalBackend) taildropEligibility(p tailcfg.NodeView) (can bool, reason string) {
```

Comment:
I think it would be good to combine this and `peerIsTaildropTargetLocked()` to make sure they use the same logic.

Reply:
Okay, after @kari-ts explained this to me, I see now that this mimics the logic in `FileTargets`, so combining it with `peerIsTaildropTargetLocked()`, as I suggested, wouldn't make sense.